ui/components/SourcesTable.tsx

```diff
@@ -42,6 +42,7 @@ function SourcesTable({ className, sources }: Props) {
 
   return (
     <URLAddressableTable
+      defaultSort={4}
```

Comment:
another comment about 4 = lastUpdated or is it status?

Reply:
There isn't really an `id` field or something, that would be nice, maybe we do in a follow up though..

Reply:
We could also make it a property of the `Field`, like `defaultSort: true` on the one we wanna start sorting by.

E.g. Then it would become:
```
  const [sortField, setSort] = React.useState(fields.find(f => f.defaultSort) || fields[0]);
```

Feels a little less fragile if we add/rm/move columns. Could also be a follow up though.